src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
```

Comment:
Really great work people! Just a few minor observations/suggestions:
It might be better to use a more descriptive name for the 'state' variable, for readability's sake.

Reply:
Maybe you could rename that piece of state into ```item```/```setItem```. Then purchaseItem could be called as ```value``` or ```type```?

Reply:
Thank you for the suggestion.